Persephone/Controllers/WindowController.swift

```diff
@@ -28,6 +33,15 @@ class WindowController: NSWindowController {
     )
   }
 
+  override func keyDown(with event: NSEvent) {
```

Comment:
Do you think it would be worth using a different method that can return a value to indicate whether the event was handled? That way the monitor block could return `nil` when the space bar press is handled.

Reply:
I don't think I'm completely sure what you're suggesting here. Are you suggesting a separate method that gets called from within `keyDown`? Or just having this method return `nil` on success?

Reply:
I was thinking having a separate helper method instead of overriding `keyDown(with:)`.

Reply:
Wouldn't I still need to override `keyDown(with:)` to capture the event?

Reply:
That comment was related to the previous event monitor approach. Feel free to ignore now!

Reply:
Aha! 👍